iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/serviceclient/ServiceClientTests.java

```diff
@@ -65,7 +65,8 @@
 {
     protected static String iotHubConnectionString = "";
     protected static String invalidCertificateServerConnectionString = "";
-    private static final String content = "abcdefghijklmnopqrstuvwxyz1234567890";
+    private static final byte[] SMALL_PAYLOAD = new byte[1024];
+    private static final byte[] LARGEST_PAYLOAD = new byte[65000]; // IoT Hub allows a max of 65 kb per message
```

Comment:
64 or 65?

Reply:
65000 is about as close to the max as we can test since there are other flags and properties that compose the AMQP frame that maxes out at 65 * 1024